sphinx_needs/directives/needimport.py

```diff
@@ -184,8 +188,13 @@ def run(self) -> Sequence[nodes.Node]:
         needs_list = needs_list_filtered
 
         # tags update
-        for need in needs_list.values():
-            need["tags"] = need["tags"] + tags
+        if tags := [
+            tag.strip()
+            for tag in re.split("[;,]", self.options.get("tags", ""))
```

Comment:
same here, the new function should be used (or is that in a different open PR?)

Reply:
this was already existing, but again no dynamic functions.
I feel you are confusing this directive with a `need` declaration directive, that is the only place we use dynamic functions in the directive options